### PR TITLE
Multiple commits which allow proper function soreport in sysadmin role

### PR DESCRIPTION
**Pull request by Koncpa**
Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1965251

### CODE REVIEW

File: policy/modules/contrib/iotop.te [hunk 1]
```diff
@@ -20,8 +20,10 @@ role iotop_roles types iotop_t;
 #
 
 allow iotop_t self:capability net_admin;
+allow iotop_t self:netlink_generic_socket { bind create getattr setopt };
```
Comment (zpytela) [inline policy/modules/contrib/iotop.te]: Please use object permissions sets when they are available, like `create_socket_perms`.

File: policy/modules/contrib/iotop.te [hunk 2]
```diff
@@ -20,8 +20,10 @@ role iotop_roles types iotop_t;
...
 allow iotop_t self:netlink_route_socket r_netlink_socket_perms;
 allow iotop_t self:netlink_socket create_socket_perms;
+allow iotop_t self:unix_dgram_socket { create getopt setopt } ;
```
Comment (zpytela) [inline policy/modules/contrib/iotop.te]: Same here.

**Review by zpytela — CHANGES_REQUESTED** [review]
Also please change "sysadm domain" to "sysadm_t domain" in one of the commit messages.

File: policy/modules/contrib/iotop.te (hunk 1 above)
Comment (Koncpa) [reply policy/modules/contrib/iotop.te]: DONE

File: policy/modules/contrib/iotop.te (hunk 2 above)
Comment (Koncpa) [reply policy/modules/contrib/iotop.te]: DONE

**Comment by zpytela** [comment]
Merging, thank you.

**Review by zpytela — APPROVED** [review]

**Pull request merged**